### PR TITLE
chore(showcase): Add Zoe Rodriguez Portfolio & TriActive USA

### PR DESCRIPTION
**Pull request by brohlson**

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -6574,3 +6574,27 @@
...
   featured: false
+- title: Zoe Rodriguez
+  url: https://zoerodrgz.com
+  main_url: https://zoerodrgz.com
+  description: >
+    Portfolio for Los Angeles-based designer Zoe Rodriguez. Built with Gatsby.
+  categories:
+    - Portfolio
+    - Design
+    - DatoCMS
```
Comment (wardpeet) [inline docs/sites.yml]: Gatsbot is complaining about DatoCMS not being valid category
```suggestion
```

File: docs/sites.yml [hunk 2]
```diff
@@ -6574,3 +6574,27 @@
...
+  built_by_url: https://chaseohlson.com
+  featured: false
+- title: TriActive USA
+  url: https://triactiveusa.com
+  main_url: https://triactiveusa.com
+  description: >
+    Website and blog for TriActive USA. Built with Gatsby.
+  categories:
+    - Landing Pages
+    - Business Website
```
Comment (wardpeet) [inline docs/sites.yml]:
```suggestion
    - Business
```

File: docs/sites.yml [hunk 3]
```diff
@@ -6574,3 +6574,27 @@
...
+    - DatoCMS
```
Comment (wardpeet) [inline docs/sites.yml]:
```suggestion
```

**Comment by brohlson** [comment]
Oops sorry @wardpeet - didn't realized the categories were capped.  I've committed your suggestions.  Let me know if that's good to go 👍

**Comment by brohlson** [comment]
Linter isn't showing any changes locally so I'm not sure why the build is failing 🤷‍♂️

**Pull request merged**